src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -794,6 +794,7 @@ public without sharing class UTIL_CustomSettingsFacade {
         allocationsSettings.Payment_Allocations_Enabled__c = mySettings.Payment_Allocations_Enabled__c;
         allocationsSettings.Rollup_N_Day_Value__c = mySettings.Rollup_N_Day_Value__c;
         allocationsSettings.Use_Fiscal_Year_for_Rollups__c = mySettings.Use_Fiscal_Year_for_Rollups__c;
+        allocationsSettings.Payment_Allocation_Two_Way_Sync_Enabled__c = true;
```

Comment:
It should grab the value from the mySettings object so we can make if true or false, as necessary.

Reply:
Please also update configAllocationsSettings to set it explicitly to false.

Reply:
Although it is not required now, if you have time to set the two way sync enabled flag to true in the two way sync tests, that would be very helpful as it would be less to do later.  Also, make sure you run all Allocation related tests, at a minimum...

Reply:
I ran all the test classes yesterday and then were all passing.

Reply:
Updated the configAllocationsSettings to set the value to false

Updated to pull the value from mySettings object